Traceback (most recent call last):
  File "../python_scripts/sonar_tf.py", line 40, in <module>
    X_train, X_test, y_train, y_test = prep_sonar_dataset('sonar_data.csv')
  File "../python_scripts/sonar_tf.py", line 14, in prep_sonar_dataset
    _sonar_df = pd.read_csv(csv_file, header=None)  #column 60 is the label
  File "/srv/home/mmalloy/.conda/envs/ece697/lib/python3.7/site-packages/pandas/io/parsers.py", line 676, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/srv/home/mmalloy/.conda/envs/ece697/lib/python3.7/site-packages/pandas/io/parsers.py", line 448, in _read
    parser = TextFileReader(fp_or_buf, **kwds)
  File "/srv/home/mmalloy/.conda/envs/ece697/lib/python3.7/site-packages/pandas/io/parsers.py", line 880, in __init__
    self._make_engine(self.engine)
  File "/srv/home/mmalloy/.conda/envs/ece697/lib/python3.7/site-packages/pandas/io/parsers.py", line 1114, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/srv/home/mmalloy/.conda/envs/ece697/lib/python3.7/site-packages/pandas/io/parsers.py", line 1891, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 374, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 674, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File sonar_data.csv does not exist: 'sonar_data.csv'
